scripts/upgrade_shapella.py

```diff
@@ -30,14 +30,18 @@
 25. Revoke `SET_BEACON_SPEC` role from `Voting`
 26. Revoke `SET_REPORT_BOUNDARIES` role from `Voting`
 27. Revoke `SET_BEACON_REPORT_RECEIVER` role from `Voting`
+28. Grant `MANAGE_TOKEN_URI_ROLE` role to `Voting`
+29. Set `WithdrawalQueueERC721` baseUri to `https://wq-api.lido.fi/v1/nft`
+30. Revoke `MANAGE_TOKEN_URI_ROLE` role from `Voting`
```

Comment:
Do we need to revoke though?

Reply:
Better to revoke. Will be sanitizing roles for the Dual Gov anyways.

Reply:
Actually, there is another way. 
We can grant the role to Agent, not Voting and leave it there. But, according to Occam razor, you don't want to multiply entities, so I'd rather clean it up.